addons/binding/org.openhab.binding.yeelight/ESH-INF/thing/wonder.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="yeelight" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	
+	<thing-type id="wonder">
+		<label>Yeelight Color LED Bulb</label>
+		<description>Yeelight Color LED Bulb</description>
+
+		<channels>
+			<channel id="brightness" typeId="brightness" />
+			<channel id="color" typeId="color" />
+			<channel id="colorTemperature" typeId="colorTemperature" />
+		</channels>
+	</thing-type>
+
+	<!-- Brightness Channel -->
+	<channel-type id="brightness">
```

Comment:
You must not define the same channel-type multiple times in a single binding - you can simply move those definitions to a channels.xml file and re-use them from all your thing types.